src/opm/output/eclipse/WriteInit.cpp

```diff
@@ -411,7 +411,8 @@ namespace {
             {
                 units.from_si(prop.unit, value);
 
-                for (auto n = dflt.size(), i = decltype(n){}; i < n; ++i) {
+                const auto n = dflt.size();
+                for (auto i = decltype(n){}; i < n; ++i) {
```

Comment:
Then it's better to write this as
```C++
for (auto n = dflt.size(), i = 0*n; i < n; ++i) {
    // ...
}
```

Reply:
sure, updated. And removed a white space.